src/main/java/collections/myHashMap/MyHashMap.java

```diff
@@ -0,0 +1,117 @@
+package collections.myHashMap;
+import collections.MyLinkedList;
+
+import java.util.LinkedList;
+import java.util.Objects;
+
+public class MyHashMap<K, V> {
+    LinkedList<Entry>[] table = new LinkedList[8];
+    public int size = 0;
+
+    public MyHashMap(){}
+
+    public void put(Key key, Value value){
+        if(size >= table.length){
+            resize();
+        }
+
+        int ix = getIndex(key) % table.length;
+
+        if(table[ix] == null){
+            table[ix] = new LinkedList<>();
+            table[ix].add(new Entry(key, value));
+            size++;
+            return;
+        }
+        else{
+            for(Entry entry : table[ix])
+                if (entry.key.equals(key)) {
+                    entry.value = value;
+                    return;
+                }
+            }
+
+        table[ix].add(new Entry(key, value));
+        size++;
+        return;
+    }
+
+    public Value get(Key key){
+        int ix = getIndex(key) % table.length;
+
+        if(table[ix] == null) {
+            return null;
+        }
+
+        for (Entry entry : table[ix]) {
+            if(entry.key.equals(key)){
+                return entry.value;
+            }
+        }
+        return null;
+    }
+
+    public void remove(Key key){
```

Comment:
Taking into account enormous exit points, how does one know if the value associated with a key was removed?